types/photoshop/index.d.ts

```diff
@@ -1,8 +1,9 @@
-// Type definitions for Photoshop JavaScript API 23.0
+// Type definitions for Photoshop JavaScript API 24.3
 // Project: https://adobe.io/photoshop/uxp
 // Definitions by:  Adobe Photoshop <https://github.com/adobe-photoshop>
 //                  Barkin Aygun <https://github.com/baaygun>
 //                  Heewoo Ahn <https://github.com/heewooa>
+//                  Jaroslav Bereza <https://github.com/jardicc>
```

Comment:
Note that this is only used for managing the review lifecycle of DT PRs, and adding yourself here will ping you for all changes to this package and let you review it.

It's fine if that's what you want (and the other maintainers agree), but if you're just looking for credit, that is handled by the git history.

Reply:
Hello, this is intentional. I created parts of that DOM myself and typings were autogenerated from the source code. In future we will likely move typings elsewhere but before we decide we want to update it here.

Also we generate documentation from source code and that has some requirements regarding JSDOC and function signatures.

Reply:
Thanks, just checking!